examples/reference/widgets/ChatEntry.ipynb

```diff
@@ -0,0 +1,218 @@
+{
+ "cells": [
+  {
```

Comment:
For some unknown reason a `Button` displays as `False`. I would expect it to display as a `Button`.

![image](https://github.com/holoviz/panel/assets/42288570/c308dd37-d357-4370-8ee4-4375c3573d6f)

```python
import panel as pn
from panel.widgets import ChatEntry

pn.extension()

ChatEntry(
    value=pn.widgets.Button(name="CLICK ME", button_type="primary"),
).servable()
```